src/sdk/cpp/lib/tests/test_iotea.cpp

```diff
@@ -322,11 +318,6 @@ TEST(iotea, Gatherer_HasTimedOut) {
             {CallToken{"a", 100}, CallToken{"b", 200}},
             {{0, false}, {100, true}, {200, true}},
         },
-        {
-            // Multiple tokens of which one doesn't have a timeout, should timeout at earliest timeout, i.e. t=100
```

Comment:
What is the new expected behavior in such cases and is test of it possible already available?